packages/xo-server-audit/src/index.js

```diff
@@ -135,6 +137,19 @@ class AuditXoPlugin {
     this._cleaners = []
     this._xo = xo
 
+    const { enabled, schedule } = defaults(staticConfig.lastHashUpload, {
```

Comment:
No!

1. don't mutate past static config
2. `lodash.defaults` is not recursive

Simply use basic default values and `??`.